stubs/cpp/build.gradle

```diff
@@ -84,4 +106,33 @@ afterEvaluate {
     tasks.generateProto.doLast {
         delete 'build/conan/src/google'
     }
+
+    def conanDir = 'build/conan'
+    def reference = 'stellarstation/stable'
+    task createPackage(type: Exec) {
+        dependsOn generateProto
+        executable 'bash'
+        args '-c', condaCommand(
+                "conan create . -s compiler=gcc -s compiler.libcxx=libstdc++ -s compiler.version=7 $reference")
+        workingDir conanDir
+    }
+
+    task uploadPackage {
+        dependsOn createPackage
+
+        doLast {
+            exec {
+                def user = rootProject.property('bintray.user')
+                def key = rootProject.property('bintray.key')
+
+                executable 'bash'
+                args '-c', condaCommand(
+                        "conan user -p $key -r stellarstation $user && " +
```

Comment:
I can't find this in my bash history or even setting environment variables, but I think we can avoid two steps by using variables

https://docs.conan.io/en/latest/reference/env_vars.html

Reply:
I switched to using env vars, but it seems to end up running 'conan user' anyway. (Conan detects the failure to authenticate, then gets the details from the env and runs conan user).